nengo_gui/config.py

```diff
@@ -4,21 +4,33 @@
 
 import nengo_gui.components
 
+def make_param(name, default):
+    try:
+        # the most recent way of making Parameter objects
+        p = nengo.params.Parameter(name=name, default=default)
+    except:
```

Comment:
This should be less general and only catch the actual exception raised for a non existing keyword argument.